code/modules/mob/living/silicon/ai/ai_defense.dm

```diff
@@ -53,3 +53,46 @@
 
 /mob/living/silicon/ai/flash_act(intensity, override_blindness_check, affect_silicon, visual, type)
 	return // no eyes, no flashing
+
+/mob/living/silicon/ai/emag_act(mob/user) //An A.I is a lot more sophisticated than a Cyborg. You might not get what you bargained for. 
+	to_chat(src, "<span class='warning'>Notice: Unexpected Int*er!f!, Callback Met at !Sk@#A(. Note: Suggest Emergency Maintenence.</span>")
+	to_chat(user, "<span class='notice'>You swipe the crytographic sequencer over [src].</span>")
+	message_admins("A.I Core [src] has been emagged by [key_name(user)].")
+	for(var/mob/living/silicon/ai)
+		switch(pick(1, 2))
+			if(1) //Paranoia is worse than actual problems.
+				to_chat(src, "<span class='warning'>Unexpected magnetic fluctuation, subset path 25#, HE*!RTS#H.</span>")
+				priority_announce("We're picking up unusual longrange sensor readings in your stations network traffic. We suggest you do a thorough investigation and file a report later.", "Unexpected Readings")
+			if(2) //We do this quietly, since they went through all the trouble of getting there, and the risk of an announcement.
+				to_chat(src, "<span class='warning'>Interference pinpointed at core path t@$wo-five-fIve-0ne-th&!(ree.")
+				SetStun(75, FALSE, TRUE)
+				sleep(20)
```

Comment:
```suggestion
				sleep(2 SECONDS)
```

Reply:
YOU CAN DEFINE IT WITH SECONDS? 
WHY HAS NO ONE EVER TOLD ME THIS?
You have saved so many years of my life with this.

Reply:
SECONDS is just a macro for (* 10)